### PR TITLE
Getting phonebox_plugin to work with netbox V3.0.2 and database change

### PR DESCRIPTION
**Pull request by dathku**
Got phonebox_plugin working with netbox V3.0.2 and added the site field as an option on the number.

### CODE REVIEW

**Review by InsaneSplash — APPROVED** [review]

**Comment by iDebugAll** [comment]
Hi @dathku. Thank you for your work. Suggested changes are helpful and relevant.
There are a several points to improve:

- Please split the changes regarding Site field and NetBox 3.x compatibility into two PRs as this is two separate improvements with different requirements and history.
- It would be great to provide backward compatibility across NetBox 2.11 and NetBox 3.0+ for the plugin.
- Number and Site relation is many-to-many by its nature. Any single number may belong to multiple Sites. The most common example is two IP-phones with the same directory number placed into two different offices. PSTN numbers may also span across multiple Sites, Regions and DCs (think of 911 or +1-800 toll-free numbers in case of NANP).

**Comment by dathku** [comment]
Thank you for the comments, I will see if I can make those changes and resubmit them!

**Comment by InsaneSplash** [comment]
note: sorry about this "InsaneSplash approved these changes 9 hours ago" wasn't intentional.

File: phonebox_plugin/navigation.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 menu_items = (
     PluginMenuItem(
         link='plugins:phonebox_plugin:list_view',
-        link_text='Numbers',
+        link_text='Numbers ',
```
Comment (chriscush765) [inline phonebox_plugin/navigation.py:6]: Is the space here significant?

**Pull request closed without merging**